docs/installation/upgrading-from-1.1.md

```diff
@@ -0,0 +1,49 @@
+# Upgrading DjNRO from 1.1 to 1.2 or later
```

Comment:
It might be worth mentioning the version that the migration is targeting.
This may reduce confusion later after several versions have passed and the context of the current master is lost

Reply:
This was written with the intention that after this PR gets merged, DjNRO 1.2 would get released.   So this doc assumes the previous migration [upgrading-from-1.0.md](upgrading-from-1.0.md) would be already applied and this would be next.

The document header says "from 1.1 to 1.2" (though filename only says "from 1.1").

How would you like to see the doc changed?

Reply:
I misread this as upgrading from any version between 1.1 and 1.2 to the current version.
I see what is intended now. No issue.